src/components/ToDo/AddCard.js

```diff
@@ -0,0 +1,187 @@
+        .map((user) => ({
+          name: user.userName,
+          id: user.userId,
+        }));
+      setAllUserInGroup(groupUserNames);
+      setIsLoading(false);
+      console.log("infinite loop warning");
+    };
+    getUserInGroup();
+  }, [selectedGroup]);
+
+  const handleDropDown = (e) => {
+    setSelectedGroup(e.target.value);
+  };
+
+  const handleSelectUser = (e) => {
+    setSelectedUser(e.target.value);
+  };
+
+  const handleSelectDate = (e) => {
+    setSelectedDate(e.target.value);
+  };
+
+  const handleTitel = (e) => {
+    setTitel(e.target.value);
+  };
+
+  const handleSave = async (e) => {
+    if (title || selectedDate) {
+      const userId = allUserInGroup.map((user) => user.userName === selectedUser).id;
+      await addListEntry(title, selectedDate, selectedGroup, userId, list);
+      setSnackbarContent({
+        message: "Dein Eintrag wurde erfolgreich erstellt.",
+        status: "success",
+        open: true,
+      });
+      close();
+      setSelectedGroup("");
+      setTitel("");
+      setSelectedDate("");
+      setSelectedUser("");
+    } else {
+      setSnackbarContent({
+        message: "Fülle bitte alle Felder aus!",
+        status: "error",
+        open: true,
+      });
+    }
+  };
+
+  return (
+    <>
+      <Snackbar snackbarContent={snackbarContent} setSnackbarContent={setSnackbarContent} />
+      <Dialog open={open} onClose={close} className={classes.dialog}>
+        <DialogTitle id="aufgabe-hinzufügen" className={classes.dialogTitle}>
+          <Typography variant="h1"> {cardTitle} hinzufügen</Typography>
+        </DialogTitle>
+        <DialogContent dividers>
+          <form className={classes.root} noValidate autoComplete="off">
+            <TextField
+              id="title field"
+              label="Titel"
+              onChange={handleTitel}
+              value={title}
+              style={{ marginBottom: 10 }}
+              className={classes.textField}
+            />
+            <TextField
+              id="date"
+              onChange={handleSelectDate}
+              label="Frist"
+              type="date"
+              value={selectedDate}
+              className={classes.textField}
+              InputLabelProps={{
+                shrink: true,
+              }}
+            />
+            <FormControl className={classes.formControl}>
+              <InputLabel htmlFor="selectGroup">Gruppe</InputLabel>
+              <Select native value={selectedGroup} onChange={handleDropDown}>
+                {!isUserData &&
+                  Object.entries(userData.groups).map((group, idx) => (
+                    <option key={idx} value={group[0]}>
+                      {group[1]}
+                    </option>
+                  ))}
+              </Select>
+            </FormControl>
+            <FormControl className={classes.formControl}>
+              <InputLabel htmlFor="selectGroup">Zuständige*r</InputLabel>
```

Comment:
```suggestion
              <InputLabel htmlFor="selectGroup">Zuständig</InputLabel>
```
Sollen wir das noch so ändern. Finde ich einfacher?